core/src/org/pentaho/platform/engine/core/system/PentahoSystem.java

```diff
@@ -1161,27 +1089,23 @@ public static String publish( final IPentahoSession session, final String classN
   }
 
   // FIXME: should be named getAdministrationPlugins
-  public static List getPublisherList() {
-    return new ArrayList( PentahoSystem.administrationPlugins );
+  public static List<IPentahoPublisher> getPublisherList() {
```

Comment:
Again, changing the return type from a public static method. This change isn't necessary for JUnit testing, and could break community publisher plugins.